.github/workflows/pr.yaml

```diff
@@ -21,11 +21,12 @@ jobs:
       - conda-python-build
       - conda-python-tests
       - docs-build
+      - wheel-build-libraft
       - wheel-build-pylibraft
       - wheel-tests-pylibraft
       - wheel-build-raft-dask
       - wheel-tests-raft-dask
-      - devcontainer
+      # - devcontainer
```

Comment:
TODO: Put this back before merging.

Reply:
Just pushed https://github.com/rapidsai/raft/pull/2531/commits/93f39fb70031bdbd1e2a865692e2baf1089e225a restoring the devcontainer builds.

They'll still fail because the manifest in devcontainers needs to be updated. Summarizing plan we agreed to offline:

1. get devcontainers PR passing CI, pointed at this branch from my fork (https://github.com/rapidsai/devcontainers/pull/438)
2. admin-merge this PR
3. point that devcontainers PR back at `rapidsai/raft`, observe it passing CI, merge it normally